### ISSUE TITLE
[Bug]: Random Maint Room has 5 2% Chance Xeno Egg Spawners in it

### ISSUE THREAD

**Issue by Programs-The-Station** [BeeStation/BeeStation-Hornet#11167]
### Round Date

N/A (Private Test Server running up to date code)

### EDIT: sk_rdm130_benoegg.dmm has 5x 2% xeno egg chance spawners in it... is this intentional?
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/100493881/3d0b444a-1959-411b-ada6-9e22155ab9cc)

Leaving original info up for context

In Corg Station, if a specific xeno maint room spawns, a live xeno egg can be delivered to it.
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/100493881/adfdd3d0-35da-4102-b446-c6849701bcc0)
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/100493881/dedf96f8-986f-486a-a36d-33a9380e2875)
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/100493881/2bff9c9a-44f1-4b03-b898-8a9e990d5956)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/100493881/f1340ea2-4ddb-4194-96b1-0fd810da30b4)

### What did you expect?

I expected the grief egg to appear in the secured pen and not in Tider's Heaven.

### Reproducible steps

1. Have Corgstation Loaded
2. Get the Large Xeno Maint Room to Spawn
3. Trigger the Xeno Egg Delivery to trigger

**Comment by EvilDragonfiend** [comment]
yes, kinda intentional for xeno, but maint one seems to be meh

**Comment by MarkusLarsson421** [comment]
Don't think I've ever seen that room, is it a recent addition?

**Comment by Programs-The-Station** [comment]
Ha, it's not recent, it's from 2020...
#2273

It is a bigger room, so doesn't spawn often.

**Comment by PowerfulBacon** [comment]
Non-issue imo, maints can be dangerous and lead to bad things happening. Bad things happening is interesting
